Toolset/palettes/script editor/behaviors/revsegutterbehavior.livecodescript

```diff
@@ -401,42 +401,28 @@ private command updateBreakpointPositions pObject, pOffset, pLine, pOldNumber, p
    
    if pOldNumber < pNewNumber then
       repeat for each line tBreakpoint in tBreakpoints
-         # In the case that the line that contained the breakpoint was where the text replacement
-         # started, we have to check for one of three options. This is slow :(
          if item -1 of tBreakpoint = pLine then
-            local tContext
-            put char 1 to pOffset of getScriptField() into tContext
-            local tNextChar
-            put char -1 of tContext into tNextChar
+            local tLineIndex
+            put the lineIndex of char pOffset of getScriptField() into tLineIndex
             
-            local tPreviousChars
-            repeat with x = the number of chars of tContext - 1 down to 1
-               if char x of tContext is empty or char x of tContext is return then
-                  exit repeat
-               end if
-               put char x of tContext after tPreviousChars
-            end repeat
+            local tCharIndex
+            put the charIndex of line tLineIndex of getScriptField() into tCharIndex
             
-            local tWhitespaceBefore
-            put matchText(tPreviousChars, "^\s*$") into tWhitespaceBefore
+            local tLine
+            put the text of line tLineIndex of getScriptField() into tLine
             
-            local tEndofLine
-            put (tNextChar is return or tNextChar is empty) into tEndofLine
+            local tBefore
+            put char 1 to (pOffset - tCharIndex) of tLine into tBefore
             
-            # This means the user has split the line containing the breakpoint, in this case we leave the breakpoint as it was
-            if (not tWhitespaceBefore) and (not tEndofLine) then
-               next repeat
-            end if
-            
-            # This means that the user has added a return to the end of the line containing the breakpoint, again we ignore this
-            #if (tPreviousChar is not return and tPreviousChar is not empty) and (tNextChar is return or tNextChar is empty) then
-            if (not tWhitespaceBefore) and tEndofLine then
-               next repeat
+            local tBeginningOfLine
+            put tBefore is empty into tBeginningOfLine
+            if not tBeginningOfLine then
+               put matchText(tBefore, "^\s*$") into tBeginningOfLine
```

Comment:
Is this faster than just `put word 1 of tBefore is empty into tBeginningOfLine`?

Reply:
I don't know... I guess it might be...

Reply:
the answer is if you do that 1000 times it's 2 milliseconds faster to use word 1 is empty. Probably not worth worrying about I suspect but I can change if you think it's necessary

Reply:
Hah :-) Well, I only suggested because it's simpler, but not a big deal

Reply:
I think the put...is empty form is easier to grok later on as well. My 2 cents worth.

Reply:
Yeah... I think I might change it

Reply:
Cool... that's .002 milliseconds I can put to use elsewhere.